### DISCUSSION TITLE
`starship init nu` creates an invalid nu file

### DISCUSSION THREAD

**Discussion by cidit** [starship/starship#5058] [Q&A] [answered]
so, my goal here is to rock alacritty + nu + starship.
i followed the steps to that intent (here)[https://starship.rs/guide/#%F0%9F%9A%80-installation].

BUT i got the following error when launching the shell:
```
Error: nu::parser::unknown_flag

  × The `term size` command doesn't have flag `-c`.
    ╭─[init.nu:10:1]
 10 │     # jobs are not supported
 11 │     let width = (term size -c | get columns | into string)
    ·                             ┬
    ·                             ╰── unknown flag
 12 │     ^/usr/local/bin/starship prompt $"--cmd-duration=($env.CMD_DURATION_MS)" $"--status=($env.LAST_EXIT_CODE)" $"--terminal-width=($width)"
    ╰────
  help: Available flags: --help(-h). Use `--help` for more information.
```
as you can imagine running `term size -c` in nushell produces the same thing.

for now my only fix is removing the last line of the nushell config (the one that regenerates the cache file every time you open your terminal using the command in the title) and manually removing the problematic flag in the script.

so, everything works for now but i find my solution pretty ducktape-y and im wondering if anyone else encountered the same problem and solved it differently.

**Comment by davidkna** [comment, marked as answer]
Nushell frequently has breaking changes which require us to make changes to starship. This issue has already been fixed in the latest version of starship, please upgrade your executable.
To warn you again, the latest version of starship isn't yet compatible with the just release v0.80.
